src/locales/de/modifier-type.ts

```diff
@@ -182,6 +182,8 @@ export const modifierType: ModifierTypeTranslationEntries = {
 
     "SOOTHE_BELL": { name: "Sanftglocke" },
 
+    "EVIOLITE": { name: "Evolith", description: "Ein mysteriöser Klumpen, der die Verteidigung und Spezial-Verteidigung von Pokémon erhöht, die sich noch entwickeln können." },
```

Comment:
```suggestion
    "EVIOLITE": { name: "Evolith", description: "Ein mysteriöser Klumpen, der die Vert. u. Spez.-Vert. von Pokémon erhöht, die sich noch entwickeln können." },
```

Try this one. Its the official description from BW/BW2. Basically says the same but just shortens the Def and SpezDef Words. (i am a german native so you can trust me on this)

Reply:
(Please try if it is short enough)

Reply:
Just tried it, fits perfectly.
<img width="675" alt="evioliteDENew" src="https://github.com/pagefaultgames/pokerogue/assets/109637146/585e54a3-93a4-4a10-9b39-ec5c5fd00d80">

Gonna commit it now, thanks!